### ISSUE TITLE
Failed to start Container Runtime Interface for OCI (CRI-O) - CentOS 8

### ISSUE THREAD

**Issue by s2000coder** [cri-o/cri-o#4030]
Followed the instructions below to install CRI-O 1.18 on CentOS 8.
```
sudo dnf -y install 'dnf-command(copr)'
sudo dnf -y copr enable rhcontainerbot/container-selinux
sudo curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_8/devel:kubic:libcontainers:stable.repo
sudo curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:1.18.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.18/CentOS_8/devel:kubic:libcontainers:stable:cri-o:1.18.repo
sudo dnf -y install cri-o
```
cri-o installed sucessfully.
```
# rpm -qi cri-o
Name        : cri-o
Epoch       : 2
Version     : 1.18.2
Release     : 2.3.el8
Architecture: x86_64
Install Date: Tue Jul 28 17:14:57 2020
Group       : Unspecified
Size        : 164007210
License     : ASL 2.0
Signature   : RSA/SHA256, Thu Jul 23 19:13:26 2020, Key ID 4d64390375060aa4
Source RPM  : cri-o-1.18.2-2.3.el8.src.rpm
Build Date  : Thu Jul 23 19:13:12 2020
Build Host  : goat19
Relocations : (not relocatable)
Vendor      : obs://build.opensuse.org/devel:kubic
URL         : https://github.com/cri-o/cri-o
Summary     : Kubernetes Container Runtime Interface for OCI-based containers
Description :
Kubernetes Container Runtime Interface for OCI-based containers
```

After installation, I get this error when starting cri-o service.
```
Jul 28 18:06:58 k8smaster systemd[1]: Starting Container Runtime Interface for OCI (CRI-O)...
Jul 28 18:06:58 k8smaster crio[3468]: time="2020-07-28 18:06:58.746075561-04:00" level=fatal msg="runtime config: conmo>
Jul 28 18:06:58 k8smaster systemd[1]: crio.service: Main process exited, code=exited, status=1/FAILURE
Jul 28 18:06:58 k8smaster systemd[1]: crio.service: Failed with result 'exit-code'.
Jul 28 18:06:58 k8smaster systemd[1]: Failed to start Container Runtime Interface for OCI (CRI-O).
```
From journalctl
```
# journalctl -xe
-- Support: https://access.redhat.com/support
--
-- Unit crio-wipe.service has begun starting up.
Jul 28 18:15:57 k8smaster crio[3482]: version file /var/run/crio/version not found: open /var/run/crio/version: no such>
Jul 28 18:15:57 k8smaster systemd[1]: Started CRI-O Auto Update Script.
-- Subject: Unit crio-wipe.service has finished start-up
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit crio-wipe.service has finished starting up.
--
-- The start-up result is done.
Jul 28 18:15:57 k8smaster systemd[1]: Starting Container Runtime Interface for OCI (CRI-O)...
-- Subject: Unit crio.service has begun start-up
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit crio.service has begun starting up.
Jul 28 18:15:57 k8smaster crio[3489]: time="2020-07-28 18:15:57.527436406-04:00" level=fatal msg="runtime config: conmo>
Jul 28 18:15:57 k8smaster systemd[1]: crio.service: Main process exited, code=exited, status=1/FAILURE
Jul 28 18:15:57 k8smaster systemd[1]: crio.service: Failed with result 'exit-code'.
Jul 28 18:15:57 k8smaster systemd[1]: Failed to start Container Runtime Interface for OCI (CRI-O).
-- Subject: Unit crio.service has failed
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit crio.service has failed.
--
-- The result is failed.
```

Any advice?

**Comment by haircommander** [comment]
I believe this is a duplicate of https://github.com/cri-o/cri-o/issues/3922

**Comment by haircommander** [comment]
actually, let's have https://github.com/cri-o/cri-o/issues/3818 be the issue to rule them all

**Comment by s2000coder** [comment]
The fix in [#3776](https://github.com/cri-o/cri-o/issues/3776 ) resolves the issue. 
Basically, replacing /usr/libexec/crio/conmon with /usr/bin/conmon in /etc/crio/crio.conf.

**Comment by haircommander** [comment]
great to hear, sorry for the problem. I hope to have the legit packaged fix coming soon

Thanks for trying out cri-o, and opening the issue. I'm closing this now, #3818 will be where the further fixes are tracked
